src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck start working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop working");

```

Comment:
Similarly, the stop message should be more descriptive. Consider 'Truck has stopped working.' to provide more clarity.